src/open-id-connect-routing.ts

```diff
@@ -21,30 +18,30 @@ export default class OpenIdConnectRouting {
     public configureRouter(routerConfiguration: RouterConfiguration) {
         this.addLoginRedirectRoute(routerConfiguration);
         this.addLogoutRedirectRoute(routerConfiguration);
-        routerConfiguration.addPipelineStep("authorize", OpenIdConnectAuthorizeStep);
+        routerConfiguration.addPipelineStep('authorize', OpenIdConnectAuthorizeStep);
     }
 
     private addLoginRedirectRoute(routerConfiguration: RouterConfiguration) {
         routerConfiguration.mapRoute({
-            name: "logInRedirectCallback",
+            name: 'logInRedirectCallback',
             navigationStrategy: (instruction: NavigationInstruction) => {
                 if (this.isSilentLogin()) {
                     return this.openIdConnectNavigationStrategies.silentSignInCallback(instruction);
                 } else {
                     return this.openIdConnectNavigationStrategies.signInRedirectCallback(instruction);
                 }
             },
-            route: this.getPath(this.openIdConnectConfiguration.redirectUri),
+            route: this.getPath(this.openIdConnectConfiguration.redirectUri!),
```

Comment:
Note-to-self: Think about the use of the bang operator here, because it might be that we can set useful defaults instead of `undefined`, thereby obviating the need for the bang operator.